src/main/resources/schema/type.graphqls

```diff
@@ -1,13 +1,58 @@
+"Custom Scalar Date"
+scalar DateTime
+
+
 type Wish {
- id: ID!
- authorId: ID!
- title: String!
- content: String!
- url: String!
- coverImageUrl: String
+  id: ID!
+  author: Wisher!
+  title: String!
+  description: String!
+  tags: String!
+  coverImageUrl: String
+  votes: [Vote]
+  comments: [Comment]
+  createdAt: DateTime
+  updatedAt: DateTime
 }
 
 type WishUpdate {
   success: Boolean!
   wish: Wish
 }
+
+interface User {
+  id: ID!
+  userName: String!
+  userProfileImage: String
+  createdAt: DateTime
+  updatedAt: DateTime
+}
+
+type Wisher implements User{
+  id: ID!
+  userName: String!
+  userProfileImage: String
+  wishList: [Wish]
+  feedTags: [String]
```

Comment:
Good idea to keep user favourite tags 👍 :)

Reply:
Thanks :smile: